libs/ui-lib/lib/cim/components/Agent/BMCForm.tsx

```diff
@@ -180,26 +180,25 @@ const emptyValues: AddBmcValues = {
   bootMACAddress: '',
   disableCertificateVerification: true, // TODO(mlibra)
   online: true,
-  nmState:
-    'interfaces:\n\
-- name: <nic1_name>\n\
-  type: ethernet\n\
-  state: up\n\
-  ipv4:\n\
-    address:\n\
-    - ip: <ip_address>\n\
-      prefix-length: 24\n\
-    enabled: true\n\
-dns-resolver:\n\
-  config:\n\
-    server:\n\
-    - <dns_ip_address>\n\
-routes:\n\
-  config:\n\
-  - destination: 0.0.0.0/0\n\
-    next-hop-address: <next_hop_ip_address>\n\
-    next-hop-interface: <next_hop_nic1_name>\n\
-',
+  nmState: `interfaces:
```

Comment:
nit: if the `interfaces:` starts on a new line, the formatting will be easier to read. But LGTM.